lib/MemoryModel/PointerAnalysis.cpp

```diff
@@ -343,7 +343,7 @@ void PointerAnalysis::printIndCSTargets(const CallICFGNode* cs, const FunctionSe
 {
     outs() << "\nNodeID: " << getFunPtr(cs);
     outs() << "\nCallSite: ";
-    cs->getCallSite()->print(outs());
+    // TODO-os cs->getCallSite()->print(outs());
```

Comment:
For all print(outs()) can be written as ``outs() << value2String(cs->getCallSite())``